CONTRIBUTING.md

```diff
@@ -32,6 +32,50 @@ The comfort and safety of Layer5 community members are our priority. Please do w
 
 To help you get started on contributing to Layer5 projects, refer to the [Layer5 Newcomers Guide](https://layer5.io/community/newcomers), which aims to make contributions easier for new folks like you! See the resources and tutorials to help you get started. Along with this we have an extensive handbook on how this community works , how folks inside the community wear different hats to run it and few of its guidelines, have a look inside the [Community Handbook](https://layer5.io/community/handbook).
 
+## Issues & Pull Requests
+
+### Creating an Issue
+
+Before **creating** an Issue i.e for `features`/`bugs`/`improvements` please follow these steps:
+
+
+1. search existing Issues before creating a new issue (look to see if the issue has already been created)
+1. if it doesn't exist create a new issue giving as much context as possible (please take note and select the correct Issue type, for example `bug`, `documentation` or `feature`)
+1. if you wish to work on the Issue once it has been triaged, please include this in your Issue description
+
+### Working on an Issue (get it assigned to you)
```

Comment:
I think you can consider shortening or totally removing the text in bracket as it is already explained in the subsequent paragraph. This is optional though.

Reply:
Thanks for the feedback, will push an update without it.